src/components/experiment-tracking/run-metadata/index.js

```diff
@@ -34,20 +46,27 @@ const RunMetadata = ({ isSingleRun, runs = [] }) => {
             className={classnames('details-metadata__run', {
               'details-metadata__run--single': isSingleRun,
             })}
-            key={run.title + i} // note: this should revert back to use gitSha once the BE returns the actual value
+            key={run.title + i} // Note: change to gitSha once BE returns the value
```

Comment:
hi, BE does return gitSha and also gitBranch

Reply:
Good call. I actually changed it to `runId`. That seems safer.